tests/test_spec.py

```diff
@@ -409,3 +413,117 @@ def test_apispec_serve_spec_preserve_order(self, app):
         response_json_docs = client.get("/api-docs/openapi.json")
         assert response_json_docs.status_code == 200
         assert response_json_docs.json["paths"] == paths
+
+
+@pytest.fixture
+def flask_cli_runner(app):
+    return app.test_cli_runner()
+
+
+@pytest.fixture
+def temp_file(tmp_path):
+    file_path = tmp_path / "".join(
```

Comment:
Why not use `tmp_path` as shown in [pytest docs](https://docs.pytest.org/en/6.2.x/tmpdir.html)?

The file name doesn't have to be random.

```py
def test_create_file(tmp_path):
    output_file = tmp_path / "openapi_file"
```